src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -202,6 +194,7 @@ export class PmdCatalogWrapper extends PmdSupport {
       }
       rulePathEntries.get(language).add(pmdJarName);
     });
+    this.logger.trace(`Added PMD Jar paths: ${rulePathEntries}`);
```

Comment:
This won't log anything helpful, because stringifying a map doesn't give you any information about its contents. You'd need to actually transform the map into a string by iterating over its contents in some way.

Reply:
Ah, you are right. I was in a hurry to leave one usage of the logger so that I won't get the unused variable error. Will revisit with the adding debug lines story.